corehq/apps/domain/templates/domain/billing_statements.html

```diff
@@ -58,6 +74,24 @@
 {% endblock %}
 
 {% block paginated_list_top %}
+    {% if request|toggle_enabled:"BULK_PAYMENTS" %}
+    <div id="currentBalanceSection">
+        <h2>{% trans 'Current Balance' %}</h2>
+        <p>
+            {% trans 'Total Due:' %}
+            {{ total_balance }}
+        </p>
+        <p>
+            <button type="button"
+                    class="btn btn-success payment-button"
```

Comment:
does `payment-button` need to be here? it's a bit confusing since there's some code that binds to the `payment-button` class, though this is ignored